server/protocols/webadmin-cli/src/test/java/org/apache/james/cli/HelpVersionCommandTest.java

```diff
@@ -75,4 +88,16 @@ void shortVersionCommandShouldWork() {
         assertThat(outputStreamCaptor.toString().trim()).isEqualTo(VERSION);
     }
 
-}
+    @Test
+    void helpDomainMessageShouldWork() {
+        new CommandLine(new WebAdminCli()).execute("help", "domain");
+        assertThat(outputStreamCaptor.toString().trim()).isEqualTo(DOMAIN_HELP_MESSAGE);
+    }
+
+    @Test
+    void suggestHelpDomainShouldWork() {
+        new CommandLine(new WebAdminCli()).execute("domain");
+        assertThat(outputStreamCaptor.toString().trim()).isEqualTo(SUGGEST_DOMAIN_HELP_MESSAGE);
+    }
+
```

Comment:
Can I type `./james-cli domain --help` and get the help only for the domain things?

Reply:
No.
1. java -jar james-cli.jar i think cause we don't use Graalvm anymore

2.  Picocli support 2 type auto gernerated help command:

- "--help" option => only 'james --help' show all help for all subcommand(domain: Domain Manage for example) but will not show help for domain's subcommand(list). 

-  help subcommand shows specific help for subcommand.Eg: 'james help domain' will show :
domain list                    Show all domains
domain create               Create a domain
....

The reason why i  must make list as a subsubcommand as above PR description. There is a solution for this complicated situation. That's if we design the CLI syntax kind of: 'james-cli domainlist' not 'james-cli domain list", we can make domainlist as a subcommand. Hence, only --help option is enough for all subcommand (cause we dont use subsubcommand anymore)